StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/OnSetterFunctionCalledMessageHandler.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  OnSetterFunctionCalledMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/9/24.
+//
+
+import Foundation
+
+struct OnSetterMessage<Values: Codable & Equatable>: Decodable {
+    let setter: String
+    let values: Values?
+}
+
+class OnSetterFunctionCalledMessageHandler<Values: Codable & Equatable>: ScriptMessageHandler<OnSetterMessage<Values>> {
+    init(setter: String, didReceiveMessage: @escaping (Values?) -> Void) {
```

Comment:
Discussed over zoom/slack –

Ideally, we don't need to use an optional type here (e.g. `Values` instead of `Values?`) and can instead alias `Values` to an optional type just for `OnExitMessageHandler`. However this change caused test failures, so we're opting to address it in a followup PR rather than this one.